### PR TITLE
AK-29078 Refactor documentation of connector_aws_vpc with template

### PR DESCRIPTION
**Pull request by spikewang**
Refactor documentation of connector_aws_vpc with template.

+ Add import section.
+ Refactor description sections.
+ Refactor examples.

### CODE REVIEW

File: docs/resources/connector_aws_vpc.md [hunk 1]
```diff
@@ -1,72 +1,75 @@
...
+* When direct inter-vpc communication is enabled, several other
+  functionalities won't work, like NAT policy, segment resource share,
+  internet-facing applications and traffic policies.
+
 
 ## Example Usage
 
+This is one minimal example to create an AWS VPC connector. One
+`alkira_segment` and `alkira_credential_aws_vpc` are always
+required. `alkira_group` is optional, but it was recommended.
```
Comment (Alshadex) [inline docs/resources/connector_aws_vpc.md:50]: but it is recommended.

Comment (spikewang) [reply docs/resources/connector_aws_vpc.md:50]: Yep, good catch !

File: docs/resources/connector_aws_vpc.md [hunk 2]
```diff
@@ -1,72 +1,74 @@
...
+  Provide AWS VPC Connector resource.
 ---
 
 # alkira_connector_aws_vpc (Resource)
 
-Manage AWS VPC Cloud Connector.
+Provide AWS VPC Connector resource.
 
-This resource could be easily used along with `terraform-provider-aws`
+This resource could be used along with provider
```
Comment (mac-alkira) [inline docs/resources/connector_aws_vpc.md:12]: could sounds like a past tense i.e. "At one time" . Its a small thing, but can seems more appropriate. 
```suggestion
This resource can be used along with provider
```

File: docs/resources/connector_aws_vpc.md [hunk 3]
```diff
@@ -1,72 +1,74 @@
...
 
-Either `vpc_cidr` or `vpc_subnet` needs to be specified for routing purpose. If `vpc_cidr` is provided, it will automatically select all associated subnets of the given VPC. Otherwise, you could select certain subnets by specifying `vpc_subnet`.
+## Routing Options
+
+Either `vpc_cidr` or `vpc_subnet` needs to be specified for routing
+purpose.  If `vpc_cidr` is provided, it will automatically select all
+associated subnets of the given VPC. Otherwise, you could select
+certain subnets by specifying `vpc_subnet`.
+
+`vpc_route_tables` could be used to adjust the routing options against
```
Comment (mac-alkira) [inline docs/resources/connector_aws_vpc.md:22]: could = can

File: docs/resources/connector_aws_vpc.md [hunk 4]
```diff
@@ -78,34 +80,73 @@ resource "alkira_connector_aws_vpc" "connector1" {
...
+
+  vpc_subnet {
+    id   = aws_subnet.vpc_subnet2.id
+    cidr = aws_subnet.vpc_subnet2.cidr_block
+  }
+}
+```
+
+To create a connector and adjust the routing to use the default
+route. There could be multiple vpc_route_table sections for additional
```
Comment (mac-alkira) [inline docs/resources/connector_aws_vpc.md:144]: could = can

**Pull request merged**